src/06-loops.js

```diff
@@ -25,7 +29,11 @@ function rangeSum(min, max) {}
  *  rangeOdd(10, 15);
  *  //> [ 15, 13, 11 ]
  */
-function rangeOdd(min, max) {}
+function rangeOdd(min, max) {
+  for (let j = max; j > min; j--){
```

Comment:
I would suggest iterating forward through an array unless we need to do otherwise